### ISSUE TITLE
Manual Search - Ignored coloring not consistent

### ISSUE THREAD

**Issue by Rouzax** [pymedusa/Medusa#4020]
Medusa Info: | Branch: master                 Commit: d6eb72d4e1019e5d1286ae6ac1d7e4b318229ebe                 Version: v0.2.1                 Database: 44.9
-- | --
Python Version: | 2.7.14 (v2.7.14:84471935ed, Sep 16 2017, 20:25:58) [MSC v.1500 64 bit (AMD64)]
SSL Version: | OpenSSL 1.0.2k  26 Jan 2017
OS: | Windows-10-10.0.14393
Locale: | nl_NL.cp1252

I have an Ignored Word `TBS` but the column of `Group` seems to also look at the case of the Ignore words while the `Release` column does not. It is purely cosmetic as the result is ignore and not snatched, see the third result.
![image](https://user-images.githubusercontent.com/4103090/38767706-f1d245b8-3fe6-11e8-96f7-47e295feb5a4.png)

**Comment by sharkykh** [comment]
Can confirm this is an issue on `develop` as well.
Relevant code:
https://github.com/pymedusa/Medusa/blob/0c8b850128aa7e6700cd1fe35fa74d9d53fce88a/medusa/search/manual.py#L308-L318
I tried making the checks case-<b><i>in</i></b>sensitive but it didn't work for me.

**Comment by Rouzax** [comment]
@medariox: if I may bump this one?

**Comment by Rouzax** [comment]
I'll add to this issue since the other one I responded in was closed already.
https://github.com/pymedusa/Medusa/issues/6344

Since a few releases the Ignore colouring is not working any more. I've cleared my cache, etc.
![image](https://user-images.githubusercontent.com/4103090/75327091-16354400-587c-11ea-9126-c6bb706160b4.png)
![image](https://user-images.githubusercontent.com/4103090/75327279-64e2de00-587c-11ea-8cab-361c320675aa.png)
Could it have something to do with the underlined ones?
